source/extensions/common/tap/admin.cc

```diff
@@ -85,10 +85,10 @@ Http::Code AdminHandler::handler(absl::string_view, Http::HeaderMap&, Buffer::In
   return Http::Code::OK;
 }
 
-Http::Code AdminHandler::badRequest(Buffer::Instance& response, absl::string_view error) {
-  ENVOY_LOG(debug, "handler bad request: {}", error);
-  response.add(error);
-  return Http::Code::BadRequest;
+Http::Code AdminHandler::badRequest(Buffer::Chunker& response, absl::string_view error) {
+  absl::string_view errorString = absl::string_view{"handler bad request: "} + error;
+  ENVOY_LOG(debug, errorString);
+  response.reportError(Http::Code::BadRequest, errorString) return Http::Code::BadRequest;
```

Comment:
also the above is a syntax error, leading me to suspect you aren't compiling/testing locally. I think it will be pretty good if you can do that before pushing to github.  CI is not a good way to iterate while developing.

The first time you compile might take a long time, but iteration should be faster once you have objects locally.

Reply:
Thanks a lot, I post just to know whether I am in the right way, I will do test at local then.